Makefile

```diff
@@ -107,9 +117,22 @@ print('Hadoop ' + spark.sparkContext._gateway.jvm.org.apache.hadoop.util.Version
 env-code:  ## Print the value of ENV_CODE environment variable
 	@echo ${ENV_CODE}
 
+.PHONY: test-dbs
+createdb :=  #unset it
+test-dbs:  ## Trigger the setup of multiple test DBs that can be reused with pytest --numprocesses. Add createdb=true to force (re-)creation of Test DBs rather than reuse.
+	pytest ${if ${createdb},--create-db,} --reuse-db --numprocesses=auto --no-cov --disable-warnings -rP -vvv --capture=no --log-cli-level=WARNING --show-capture=log 2> /dev/null 'usaspending_api/tests/integration/test_setup_of_test_dbs.py::test_trigger_test_db_setup'
+
+.PHONY: test-spark-deps
+test-spark-deps:  ## Trigger a singular test in one pytest session that does nothing but cause Maven dependencies to be downloaded and cached through Ivy; reduces contention when parallel spark builds need the depdencies
+	pytest --no-cov --disable-warnings -r=fEs --verbosity=3 'usaspending_api/tests/integration/test_setup_of_spark_dependencies.py::test_preload_spark_jars'
+
 .PHONY: tests
```

Comment:
Why doesn't this service reference the requirements-dev.txt service?

Reply:
Added `local-dev-setup` as a dependency, which itself calls `requiremets-dev`. 

It depends how far we want to go with this useful shortcut command
- a) just call it and get right to tests, assuming the developer has setup all prerequisites
- b) call it and make sure it sets up all the things (virtual env, pip installs, etc.) that are needed to successfully run tests

We can try it with the dependent setup baked in, but it does add a bit of lag to getting the tests started. May be good for first-time developers on the team but experienced developers may not want that lag in their workflow. We'll see what kind of feedback we get.